cob_twist_controller/include/cob_twist_controller/cob_twist_controller_data_types.h

```diff
@@ -78,6 +83,18 @@ struct JointStates
     KDL::JntArray last_q_dot_;
 };
 
+struct ActiveCartesianDimension {
```

Comment:
If this is just a "marker" struct to set a dimension active or not then use unsigned char here with 0 / 1 entry. 

Otherwise if the cartesian velocities are meant here then name the struct like "CartesianVelocity"!